deployments/terraform_modules/santa_api/lambda.tf

```diff
@@ -57,7 +64,6 @@ module "xsrf_function" {
   lambda_source_bucket      = aws_s3_bucket_object.santa_api_source.bucket
   lambda_source_key         = aws_s3_bucket_object.santa_api_source.key
   lambda_source_hash        = local.lambda_source_hash
-  lambda_handler            = "api"
```

Comment:
since handlers for `al2.provided` images must just be called `bootstrap` this var is no longer needed